images/github-runners-manager/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:latest
+
+
+RUN set -eux ; \
+    apt update ; \
+    apt -y upgrade ; \
+    apt -y install \
+            git \
+            openssh-client \
+            python3 \
+            python3-pip \
+            curl  \
+            dbus-user-session  \
+            uidmap ; \
+# Install docker
+    curl -sSL https://get.docker.com/ | sh ;
```

Comment:
Maybe here we should note that we install docker inside this image, but only use the docker CLI (but not dockerd). The dockerd is provided outside of the container (which is provided by `/run/user/${UID}/docker.sock:/var/run/docker.sock:rw`)

Reply:
for me this is obvious. But OK to add a comment